packages/scope-sdk/src/Scope.ts

```diff
@@ -58,6 +58,22 @@ export const ScopePairs = [
   'UXP/USD',
   'kUXDUSDCOrca/USD',
   'JITOSOL/SOL',
+  'SOLEma/USD',
+  'ETHEma/USD',
+  'SRMEma/USD',
+  'BTCEma/USD',
+  'RAYEma/USD',
+  'FTTEma/USD',
+  'MSOLEma/USD',
+  'BNBEma/USD',
```

Comment:
How come you also added a bunch of other pairs here? Is this related to BONK?

Reply:
Not related, but the index matter IIRC. Those are in scope even if unused in the SDK, I added them so I have the right index.